### ISSUE TITLE
Failed adding Chart to worksheet

### ISSUE THREAD

**Issue by dfinke** [dfinke/ImportExcel#377]
`WARNING: Failed adding Chart to worksheet 'System.Collections.Hashtable.name': You cannot call a method on a null-valued expression.`

https://github.com/dfinke/ImportExcel/blob/master/Examples/Charts/MultipleCharts.ps1

This works with the 5.0.1 version in the gallery.

We need to figure out how to coordinate looking into these. My thinking is you know these parts best at the moment after the refactoring. Let me know what you think.

I wanted to do a sweep on the examples and add a line to figure out if ImportExcel was installed, if not, load it from the $PSScriptRoot. So I can test examples from my local repo

**Comment by jhoneill** [comment]
This is a screw-up of mine. I managed to check something into my repo while the last PR was open which shouldn't have been checked in.  
I've been adding chart tests this afternoon and found there are two errors at lines 825 and 826
```
 $chardef.PSObject.Properties | ForEach-Object {if ($_.value -ne $null) {$params[$_.name] = $_.value}}
Add-ExcelChart $params
```
The first line should be` $charTdef` and the second should be` @params `
I wanted to add something to charts before sending you another PR. I'm rolling some of the existing examples into the into the tests  Examples should be tested and good tests should work as examples; so the more overlap the better.

**Comment by dfinke** [comment]
Cool. Do you want me to wait it make the changes?

**Comment by jhoneill** [comment]
I've just committed the fix and sent you a PR. Also fixed #375
